share/completions/rsync.fish

```diff
@@ -84,29 +121,43 @@ complete -c rsync -l include -d "Don’t exclude files matching PATTERN"
 complete -c rsync -l port -d "Specify double-colon alternate port number"
 complete -c rsync -l sockopts -d "Specify custom TCP options"
 complete -c rsync -l blocking-io -d "Use blocking I/O for the remote shell"
+complete -c rsync -l outbuf -xa '{N,L,B}' -d "Set out buffering to None, Line, or Block"
 complete -c rsync -l stats -d "Give some file-transfer stats"
 complete -c rsync -s 8 -l 8-bit-output -d "Leave high-bit chars unescaped in output"
 complete -c rsync -s h -l human-readable -d "Output numbers in a human-readable format"
 complete -c rsync -l progress -d "Show progress during transfer"
 complete -c rsync -s P -d "Same as --partial --progress"
 complete -c rsync -s i -l itemize-changes -d "Output a change-summary for all updates"
-complete -c rsync -l log-format -x -d "Output filenames using the specified format"
-complete -c rsync -l password-file -x -d "Read password from FILE"
+complete -c rsync -s M -l remote-option -d "Send OPTION to the remote side only"
+complete -c rsync -l out-format -x -d "Output updates using the specified FORMAT"
+complete -c rsync -l log-file -d "log what we're doing to the specified FILE"
+complete -c rsync -l log-file-format -x -d "log updates using the specified FMT"
+complete -c rsync -l password-file -r -d "Read password from FILE"
+complete -c rsync -l early-input -d "Use FILE for daemon's early exec input"
 complete -c rsync -l list-only -d "List the files instead of copying them"
-complete -c rsync -l bwlimit -x -d "Limit I/O bandwidth; KBytes per second"
-complete -c rsync -l write-batch -x -d "Write a batched update to FILE"
+complete -c rsync -l bwlimit -xa '(seq 0 10){K,M,G}' -d "Limit I/O bandwidth; optional unit (KB/s default)"
+complete -c rsync -l stop-after -x -d "Stop rsync after MINS minutes have elapsed"
+complete -c rsync -l stop-at -x -d "Stop rsync at the specified point in time"
+complete -c rsync -l write-batch -r -d "Write a batched update to FILE"
 complete -c rsync -l only-write-batch -d "Like --write-batch but w/o updating dest"
-complete -c rsync -l read-batch -x -d "Read a batched update from FILE"
+complete -c rsync -l read-batch -r -d "Read a batched update from FILE"
 complete -c rsync -l protocol -x -d "Force an older protocol version to be used"
+complete -c rsync -l iconv -x -d "Request charset conversion of filenames"
 complete -c rsync -l checksum-seed -x -d "Set block/file checksum seed (advanced)"
 complete -c rsync -s 4 -l ipv4 -d "Prefer IPv4"
 complete -c rsync -s 6 -l ipv6 -d "Prefer IPv6"
-complete -c rsync -l version -d "Display version and exit"
-complete -c rsync -l help -d "Display help and exit"
+complete -c rsync -l daemon -d "Run as an rsync daemon"
+complete -c rsync -l config -r -d "Specify alternate rsyncd.conf file"
+complete -c rsync -l dparam -x -d "Override global daemon config parameter"
+complete -c rsync -l no-detach -x -d "Do not detach from the parent"
+complete -c rsync -s V -l version -d "Display version + other info and exit"
```

Comment:
Maybe "version and feature info"? It lists the sort of stuff you'd en/disable via compile flags, like the compression and checksum algorithms.

Reply:
Good idea, changed now.